eng/common/cross/build-rootfs.sh

```diff
@@ -422,7 +422,7 @@ case "$__AlpineVersion" in
         elif [[ "$__AlpineArch" == "riscv64" ]]; then
             __AlpineLlvmLibsLookup=1
             __AlpineVersion=edge # minimum version with APKINDEX.tar.gz (packages archive)
-        elif [[ -n "$__AlpineVersion" ]]; then
+        elif [[ -n "$__AlpineVersion" && "$__AlpineVersion" != "." ]]; then
```

Comment:
I think this would work too and be simpler.

```suggestion
        elif [[ -n "$__AlpineMajorVersion" ]]; then
```